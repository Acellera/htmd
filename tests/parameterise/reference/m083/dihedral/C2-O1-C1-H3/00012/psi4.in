set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.240291	 0.559484	 0.604969
C	 -2.008299	 -0.205926	 -0.129530
O	 -0.774054	 0.001095	 -0.743181
H	 -2.752932	 -0.160639	 -0.912984
H	 -2.076115	 -1.178578	 0.348530
C	 0.410107	 -0.035494	 0.033322
C	 1.525938	 -0.026612	 -1.009200
C	 0.522069	 1.202474	 0.928917
C	 0.484712	 -1.318251	 0.867639
H	 1.459897	 0.867364	 -1.619017
H	 1.433489	 -0.887043	 -1.662259
H	 2.502563	 -0.052988	 -0.536793
H	 0.396340	 2.104712	 0.339692
H	 1.497994	 1.237150	 1.402171
H	 -0.221184	 1.204606	 1.718934
H	 0.333175	 -2.186784	 0.235390
H	 1.458733	 -1.404507	 1.338144
H	 -0.259059	 -1.336856	 1.657000

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		2 3 6 7
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "18  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
