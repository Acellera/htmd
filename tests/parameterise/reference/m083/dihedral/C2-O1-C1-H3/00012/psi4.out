    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:44:02 2016

    Process ID:  13268
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.240291	 0.559484	 0.604969
C	 -2.008299	 -0.205926	 -0.129530
O	 -0.774054	 0.001095	 -0.743181
H	 -2.752932	 -0.160639	 -0.912984
H	 -2.076115	 -1.178578	 0.348530
C	 0.410107	 -0.035494	 0.033322
C	 1.525938	 -0.026612	 -1.009200
C	 0.522069	 1.202474	 0.928917
C	 0.484712	 -1.318251	 0.867639
H	 1.459897	 0.867364	 -1.619017
H	 1.433489	 -0.887043	 -1.662259
H	 2.502563	 -0.052988	 -0.536793
H	 0.396340	 2.104712	 0.339692
H	 1.497994	 1.237150	 1.402171
H	 -0.221184	 1.204606	 1.718934
H	 0.333175	 -2.186784	 0.235390
H	 1.458733	 -1.404507	 1.338144
H	 -0.259059	 -1.336856	 1.657000

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		2 3 6 7
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "18  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-1-4-3.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:44:03 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H         -2.244582113253     0.625689105335     0.612424693931     1.007825032070
           C         -2.012590113253    -0.139720894665    -0.122074306069    12.000000000000
           O         -0.778345113253     0.067300105335    -0.735725306069    15.994914619560
           H         -2.757223113253    -0.094433894665    -0.905528306069     1.007825032070
           H         -2.080406113253    -1.112372894665     0.355985693931     1.007825032070
           C          0.405815886747     0.030711105335     0.040777693931    12.000000000000
           C          1.521646886747     0.039593105335    -1.001744306069    12.000000000000
           C          0.517777886747     1.268679105335     0.936372693931    12.000000000000
           C          0.480420886747    -1.252045894665     0.875094693931    12.000000000000
           H          1.455605886747     0.933569105335    -1.611561306069     1.007825032070
           H          1.429197886747    -0.820837894665    -1.654803306069     1.007825032070
           H          2.498271886747     0.013217105335    -0.529337306069     1.007825032070
           H          0.392048886747     2.170917105335     0.347147693931     1.007825032070
           H          1.493702886747     1.303355105335     1.409626693931     1.007825032070
           H         -0.225475113253     1.270811105335     1.726389693931     1.007825032070
           H          0.328883886747    -2.120578894665     0.242845693931     1.007825032070
           H          1.454441886747    -1.338301894665     1.345599693931     1.007825032070
           H         -0.263350113253    -1.270650894665     1.664455693931     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.14715  B =      0.09155  C =      0.09144 [cm^-1]
  Rotational constants: A =   4411.44446  B =   2744.73405  C =   2741.21287 [MHz]
  Nuclear repulsion =  272.305864223499782

  Charge       = 0
  Multiplicity = 1
  Electrons    = 50
  Nalpha       = 25
  Nbeta        = 25

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 60
    Number of basis function: 114
    Number of Cartesian functions: 114
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        114     114       0       0       0       0
   -------------------------------------------------------
    Total     114     114      25      25      25       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 252
    Number of basis function: 786
    Number of Cartesian functions: 786
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 3.2332452966E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -178.64702439436991   -1.78647e+02   9.78047e-02 
   @DF-RHF iter   2:  -220.54230464837320   -4.18953e+01   4.77512e-02 DIIS
   @DF-RHF iter   3:  -241.55913674713014   -2.10168e+01   5.36565e-02 DIIS
   @DF-RHF iter   4:  -263.42398921486989   -2.18649e+01   3.37236e-02 DIIS
   @DF-RHF iter   5:  -268.96541207906745   -5.54142e+00   1.79560e-02 DIIS
   @DF-RHF iter   6:  -270.71755863400728   -1.75215e+00   8.10497e-03 DIIS
   @DF-RHF iter   7:  -271.14398625760742   -4.26428e-01   2.01711e-03 DIIS
   @DF-RHF iter   8:  -271.17198558282433   -2.79993e-02   6.26935e-04 DIIS
   @DF-RHF iter   9:  -271.17492307775507   -2.93749e-03   1.30839e-04 DIIS
   @DF-RHF iter  10:  -271.17510186732517   -1.78790e-04   4.99567e-05 DIIS
   @DF-RHF iter  11:  -271.17513098815408   -2.91208e-05   1.94445e-05 DIIS
   @DF-RHF iter  12:  -271.17513522306723   -4.23491e-06   3.75573e-06 DIIS
   @DF-RHF iter  13:  -271.17513542632952   -2.03262e-07   8.05310e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.546837     2A    -11.281274     3A    -11.260518  
       4A    -11.209724     5A    -11.207753     6A    -11.203101  
       7A     -1.370118     8A     -1.073008     9A     -0.946702  
      10A     -0.939091    11A     -0.912296    12A     -0.752237  
      13A     -0.655598    14A     -0.644475    15A     -0.637259  
      16A     -0.592411    17A     -0.575131    18A     -0.552449  
      19A     -0.534008    20A     -0.523837    21A     -0.514642  
      22A     -0.487331    23A     -0.486523    24A     -0.439058  
      25A     -0.404218  

    Virtual:                                                              

      26A      0.219073    27A      0.265105    28A      0.277899  
      29A      0.288768    30A      0.305563    31A      0.310969  
      32A      0.322027    33A      0.337430    34A      0.353315  
      35A      0.357201    36A      0.363289    37A      0.367389  
      38A      0.390172    39A      0.403084    40A      0.426929  
      41A      0.441183    42A      0.476077    43A      0.701908  
      44A      0.711189    45A      0.732604    46A      0.767975  
      47A      0.773320    48A      0.810252    49A      0.822760  
      50A      0.879289    51A      0.904602    52A      0.941277  
      53A      0.951872    54A      0.975442    55A      1.012186  
      56A      1.062649    57A      1.104308    58A      1.121234  
      59A      1.150863    60A      1.155653    61A      1.171214  
      62A      1.184970    63A      1.193758    64A      1.198132  
      65A      1.214026    66A      1.224817    67A      1.227387  
      68A      1.260983    69A      1.273804    70A      1.288633  
      71A      1.300854    72A      1.439439    73A      1.461831  
      74A      1.614384    75A      1.633719    76A      1.660433  
      77A      1.676866    78A      1.727893    79A      1.869329  
      80A      1.934317    81A      2.021658    82A      2.025809  
      83A      2.051677    84A      2.106425    85A      2.179297  
      86A      2.190431    87A      2.254418    88A      2.316713  
      89A      2.396420    90A      2.407775    91A      2.455782  
      92A      2.471147    93A      2.497986    94A      2.528973  
      95A      2.580750    96A      2.594469    97A      2.614183  
      98A      2.624116    99A      2.634445   100A      2.694239  
     101A      2.784281   102A      2.800332   103A      2.892053  
     104A      2.962007   105A      3.087247   106A      3.121733  
     107A      3.278340   108A      3.341095   109A      4.397676  
     110A      4.639352   111A      4.691824   112A      4.739718  
     113A      4.760367   114A      5.092982  

    Final Occupation by Irrep:
              A 
    DOCC [    25 ]

  Energy converged.

  @DF-RHF Final Energy:  -271.17513542632952

   => Energetics <=

    Nuclear Repulsion Energy =            272.3058642234997819
    One-Electron Energy =                -907.4628167379423758
    Two-Electron Energy =                 363.9818170881131891
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -271.1751354263293479

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.3848      Y:    -0.4118      Z:     2.8119

  Electronic Dipole Moment: (a.u.)
     X:    -1.4052      Y:     0.3083      Z:    -2.3019

  Dipole Moment: (a.u.)
     X:    -0.0204      Y:    -0.1035      Z:     0.5100     Total:     0.5208

  Dipole Moment: (Debye)
     X:    -0.0519      Y:    -0.2630      Z:     1.2963     Total:     1.3238


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-1-4-3.cx1.hpc.ic.ac.uk at Tue Jul  5 16:44:13 2016
Module time:
	user time   =      10.21 seconds =       0.17 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =         10 seconds =       0.17 minutes
Total time:
	user time   =      10.21 seconds =       0.17 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =         10 seconds =       0.17 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     1.3848      Y:    -0.4118      Z:     2.8119

  Electronic Dipole Moment: (a.u.)
     X:    -1.4052      Y:     0.3083      Z:    -2.3019

  Dipole Moment: (a.u.)
     X:    -0.0204      Y:    -0.1035      Z:     0.5100     Total:     0.5208

  Dipole Moment: (Debye)
     X:    -0.0519      Y:    -0.2630      Z:     1.2963     Total:     1.3238

  Quadrupole Moment: (Debye Ang)
    XX:   -37.1359     YY:   -40.0536     ZZ:   -41.4762
    XY:     0.4141     XZ:    -0.7841     YZ:     0.1551

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     2.4193     YY:    -0.4983     ZZ:    -1.9210
    XY:     0.4141     XZ:    -0.7841     YZ:     0.1551

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.43039  0.43039  0.00000  0.13923
       2     C     3.07964  3.07964  0.00000 -0.15929
       3     O     4.31877  4.31877  0.00000 -0.63755
       4     H     0.41285  0.41285  0.00000  0.17431
       5     H     0.42836  0.42836  0.00000  0.14328
       6     C     2.84167  2.84167  0.00000  0.31665
       7     C     3.23282  3.23282  0.00000 -0.46565
       8     C     3.24406  3.24406  0.00000 -0.48813
       9     C     3.25176  3.25176  0.00000 -0.50352
      10     H     0.41382  0.41382  0.00000  0.17235
      11     H     0.41464  0.41464  0.00000  0.17073
      12     H     0.42528  0.42528  0.00000  0.14945
      13     H     0.41314  0.41314  0.00000  0.17371
      14     H     0.42092  0.42092  0.00000  0.15815
      15     H     0.42084  0.42084  0.00000  0.15831
      16     H     0.41348  0.41348  0.00000  0.17304
      17     H     0.41823  0.41823  0.00000  0.16354
      18     H     0.41932  0.41932  0.00000  0.16137

   Total alpha = 25.00000, Total beta = 25.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
